runtime/src/snapshot_utils.rs

```diff
@@ -1722,12 +1823,19 @@ pub fn package_process_and_archive_full_snapshot(
         None,
     )?;
 
-    process_and_archive_snapshot_package_pre(
+    let package = process_and_archive_snapshot_package_pre(
         package,
         thread_pool,
         None,
         maximum_snapshots_to_retain,
-    )
+    )?;
+
+    Ok(FullSnapshotArchiveInfo::new(
+        package.tar_output_file,
+        package.slot,
+        package.hash,
+        package.archive_format,
+    ))
```

Comment:
can we embed  `SnapshotArchiveInfo` inside `AccountsPackage` (replacing these 4 fields `tar_output_file, slot, hash, archive_format`, then this call could look like 

```
Ok(FullSnapshotArchiveInfo::new(package.snapshot_archive_info))
```

Reply:
Ooo I love it! Will do.

Reply:
OK, done!